pydm/widgets/drawing.py

```diff
@@ -683,97 +591,358 @@ def flipMidPointArrow(self, new_selection):
         -------
         new_selection : bool
         """
-        if self._mid_point_arrow_flipped != new_selection:
-            self._mid_point_arrow_flipped = new_selection
+        if self._arrow_mid_point_flipped != new_selection:
+            self._arrow_mid_point_flipped = new_selection
             self.update()
 
+    @staticmethod
+    def _arrow_points(startpoint, endpoint, height, width):
```

Comment:
would it be possible to add type hints to the methods that you have added in this PR? I figured we can slowly add type hints to pydm as the code is updated